lib/kanal/plugins/batteries_bridge/bridges/telegram_bridge.rb

```diff
@@ -63,22 +63,38 @@ def setup
               val
             end
 
-            output_convert :keyboard, :tg_reply_markup do |keyboard_object|
+            output_convert :keyboard, :tg_reply_markup do |keyboard_object, input, output|
               nil if keyboard_object.nil? || !keyboard_object.to_a.count.positive?
 
-              inline_keyboard = []
+              if output.specifics.has?(:tg_reply_keyboard) == false || output.specifics.get(:tg_reply_keyboard) == false
```

Comment:
Check for :tg_reply_keyboard == true
in this case - use reply keyboard

Otherwise, use inline keyboard by default

And only one check for get == nil is sufficient

Reply:
https://github.com/idchlife/kanal-plugins-batteries_bridge/pull/8/commits/1d778544fd88c6c97062983a1905429dde9f2fdb